plugins/newsdownloader.koplugin/main.lua

```diff
@@ -586,7 +586,11 @@ function NewsDownloader:processFeed(feed_type, feeds, cookies, limit, download_f
         local feed_description
         if feed_type == FEED_TYPE_RSS then
             feed_title = feed.title
-            feed_description = feed.description[1] or feed.description --- @todo This should select the one with type="html" if there is a choice.
```

Comment:
You can write it as feed.description and feed.description[1] or feed.description. Having the title displayed as content under the title repeating it strikes me as a bit silly. :-)

Reply:
PS Best to apply the same fix a few lines down for Atom.

Reply:
Hum, that's true, I tried to keep the same style as these lines below:
```
              if feed["content:encoded"] ~= nil then
                  -- Spec: https://web.resource.org/rss/1.0/modules/content/
                  feed_description = feed["content:encoded"]
```
But your one-liner is cleaner. I'll update the PR.

> Having the title displayed as content under the title repeating it strikes me as a bit silly. :-)

I didn't want to set it to nil, I thought it would blow up without a description? Let's have a look.

Reply:
If it does you can set it to `""`.

Reply:
Yep, it blew up... We could add a `feed.description or "No description available"` somewhere

Reply:
Ok, I have pushed a new version. I am not too sure about inlining the `feed_description or "No description available",` inside the method call but....

Reply:
Yeah, How do I handle that new Message string? Should I create a new entry in the localization files? I will just set it to empty string for now and if needed we can go the extra mile and request translations

Reply:
In this case I'm of the opinion that if it's empty it's empty. ;-)

For translation in general you can simply surround the string by _() and it'll be picked up automatically, provided it's of the form of `""` or `[[]]`.